tests/unit/test_gitlab_http_methods.py

```diff
@@ -372,6 +372,63 @@ def response_callback(
     assert "http://example.com/api/v4/user/status" in error_message
 
 
+def test_http_request_on_409_resource_lock_retries(gl_retry):
+    url = "http://localhost/api/v4/user"
+    retried = False
+
+    def response_callback(
+        response: requests.models.Response,
```

Comment:
Is there a difference between `requests.modles.Response` and `requests.Response`?

Reply:
Not from my perspective, I'm ok either way! But this is how it is already used in `tests/unit/test_gitlab_http_methods.py` so this was just consistent with the other occurrences. I didn't add the initial code in the other files, I'm guessing IDEs auto-complete it where it is defined when they try to auto-detect the type.

Reply:
Thank you for the clarification